exercises/content-curation.md

```diff
@@ -0,0 +1,28 @@
+### Exercise
+
+This exercise is to assess your detective skills, and your content curation skills. It is similar to a typical task you would work on at Snyk.
+
+You are given a CVE ID and asked to discover all information on it. You do not need prior knowledge of security exploits, but it may help.
+
+You can use any external source available on the internet, NOT including Snyk.io .
+
+#### Tasks
+A large company, Equifax, was breached by hackers on March 13th, 2017. They used a known exploit, assigned the ID: **CVE-2017-5638**.
+
+1. Build a timeline of when the vulnerability was initially found, to the day Equifax was hacked.
+We'd like to know any small piece of information, including to but not limited to:
+- What Open Source package is vulnerable? Talk to us in [maven group ids and artifact ids](https://maven.apache.org/guides/mini/guide-naming-conventions.html).
+- When was the vulnerability initially published? Where? _Are you sure_ ?
```

Comment:
The `_Are you sure_?` part is a bit discouraging for me... Sending me the message I don't stand a chance to get this right. Maybe it's just me though :)